File: epub/ruby/read/epub_read.rb
#! /usr/bin/ruby
# -*- coding: utf-8 -*-
#
#	epub_read.rb
#
#					Sep/17/2014
#
# ---------------------------------------------------------------------
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
load '/var/www/data_base/common/ruby_common/epub_manipulate.rb'
# ---------------------------------------------------------------------
file_epub = ARGV[0]
puts "*** 開始 ***"
#
dict_aa = epub_to_dict_proc(file_epub)
dict_display_proc(dict_aa)
#
puts "*** 終了 ***"
# ---------------------------------------------------------------------

File: epub/python/read/epub_read.py
#! /usr/bin/python
# -*- coding: utf-8 -*-
#
#	epub_read.py
#
#						Jul/07/2012
#
import	sys
sys.path.append ('/var/www/data_base/common/python_common')
from text_manipulate import dict_display_proc
from epub_manipulate import epub_to_dict_proc
# ------------------------------------------------------------------
print	("*** 開始 ***")
file_epub = sys.argv[1]
dict_aa = epub_to_dict_proc (file_epub)
#
dict_display_proc (dict_aa)
print	("*** 終了 ***")
# ------------------------------------------------------------------

